src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Export passwords shared with me'     : 'Exporter les mots de passe partagé avec moi',
+            'Don\'t edit passwords shared with me': 'Ne pas éditer les mots de passe partagé avec moi',
+            'Run Export'                          : 'Lancer l’export',
+            'Export'                              : 'Export',
+            'Waiting...'                          : 'En attente…',
+            'Export error'                        : 'Erreur lors de l’export',
+            'Download {format}'                   : 'Télécharger le ficher {format}',
+            'Nothing to export'                   : 'Rien à exporter',
+            'There is no data to export'          : 'Il n’y a aucune donnée à exporter',
+            'Restore or import'                   : 'Restauration/Import',
+            'Passwords CSV'                       : 'CSV des mots de passe',
+            'Folder CSV'                          : 'CSV des dossiers',
+            'Tags CSV'                            : 'CSV des étiquettes',
+            'Custom CSV'                          : 'CSV personnalisé',
+            'Select File'                         : 'Sélectionner un ficher',
+            'Select Options'                      : 'Choisissez les options',
+            'Conflict handling'                   : 'Gestion des conflits',
+            'Skip if same revision'               : 'Ignorer si n° de version identique',
+            'Skip always'                         : 'Toujours ignorer',
+            'Overwrite existing'                  : 'Écraser l’existant',
+            'Merge with existing'                 : 'Fusionner avec l’existant',
+            'Create new entry'                    : 'Créer une nouvelle entrée',
+            'For encrypted backups'               : 'Nécessaire pour les sauvegardes chiffrées',
+            'CSV Options'                         : 'Options CSV',
+            'Database'                            : 'Base de donnée',
+            'Quote Character'                     : 'Séparateur de chaîne de caractères',
+            'Escape Character'                    : 'Caractère d’échappement',
+            'Field delimiter'                     : 'Séparateur de champ',
+            'Field Delimiter'                     : 'Séparateur de champ',
+            'Quote'                               : 'Apostrophe',
+            'Single Quote'                        : 'Apostrophe simple',
+            'Backslash'                           : 'Barre oblique inversée',
+            'Detect'                              : 'Detecter automatiquement',
+            'Comma'                               : 'Virgule',
+            'Semicolon'                           : 'Point-virgule',
+            'Space'                               : 'Espace',
+            'Tab'                                 : 'Tabulation',
+            'Detect unescaped quotes'             : 'Détecté les apostrophes non échappés',
```

Comment:
Should be `Détecter les apostrophes non échappées` (`é` => `er` and `és` => `ées`)